modules/store/src/action_group_creator_models.ts

```diff
@@ -0,0 +1,142 @@
+import {
+  ActionCreator,
+  ActionCreatorProps,
+  Creator,
+  FunctionWithParametersType,
+  NotAllowedCheck,
+} from '@ngrx/store';
+import { TypedAction } from './models';
+
+// Generating API docs for the `createActionGroup` function is solved by moving its types
```

Comment:
There was the same bug with generating API docs for the `createActionGroup` function as there was for the `createFeature` - template literal types. I moved its types to another file to make API generation work.